terraform/modules/ecs/main.tf

```diff
@@ -13,6 +13,7 @@ locals {
     polytomic_port = var.polytomic_port,
     env = {
       ROOT_USER                        = var.polytomic_root_user,
+      LOCAL_DATA_PATH                  = var.polytomic_data_path,
```

Comment:
I guess `LOCAL_DATA=true` is set in the environment elsewhere? I guess if the preflight worked it _must_ be :).

Reply:
Oh interesting. I set this in order to get the preflight log to output correctly. I suppose we also need to add the `LOCAL_DATA` boolean if we want syncs to actually use it